CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 # CHANGELOG
 
-# Version 3.0.1 GA
-* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp
+# Version 3.0.1-BETA
+* Friendly error messages thrown for the following scenarios
+    - Missing connection string
+    - Invalid SSL certificate issues when not able to connect to IngestionEndPoint Url, Live endpoint Url and CdsProfiler endpoint url.
+    - Invalid Telemetry Processor Configuration
+* Telemetry processor config throws null pointer exception when attribute value is not provided and matchType is regexp.
+* Map service.version to application_Version(Resolves [#1392](https://github.com/microsoft/ApplicationInsights-Java/issues/1392))
+* This release also brings interoperability with 2.x SDK:
+    - ThreadContext.getRequestTelemetryContext().getHttpRequestTelemetry().getProperties().put("key1", "val1")
```

Comment:
we can delete line 11 & 12. doc will have the details.  might be helpful to paste the link to the doc here.

Reply:
@trask can you review this?